build.gradle

```diff
@@ -58,58 +52,60 @@ sonarRunner {
     }
 }
 
-def grgit = Grgit.open(project.file('.'))
-def tags = grgit.tag.list()
-def versions = []
+afterEvaluate {
+}
 
-tags.each { Tag t ->
-    try {
-        Version version = new Version.Builder(t.name).build()
-        versions << version
-    } catch (UnexpectedCharacterException e) {
-        // Skip, tag is invalid
+task setVersionInBashScript << {
+    ant.replace(file: "bin/minimesos", token: "MINIMESOS_TAG=\"latest\"", value: "MINIMESOS_TAG=\"${project.version}\"")
+    exec {
+        workingDir "${project.rootDir}"
+        commandLine "git", "commit", "-a", "-m", "[Release ${project.version}] pre-tag commit"
+    }
+    exec {
+        workingDir "${project.rootDir}"
+        commandLine "git", "push"
     }
 }
 
-Version currentVersion = new Version.Builder("0.0.0").build()
+task resetVersionInBashScript << {
+    // have to use regular expression because the project version has changed and the previous is not kept
+    ant.replaceregexp(file: "bin/minimesos", match: "MINIMESOS_TAG=\".*\"", replace: "MINIMESOS_TAG=\"latest\"")
```

Comment:
I don't think it's likely to have multiple double quotes on one line, but I usually prefer non-greedy match
```.*?``` to terminate at the first match